changelog/_template.rst

```diff
@@ -0,0 +1,45 @@
+.. _vp{{ versiondata.version.replace(".", "p") }}:
+
+{% if top_line %}
+{{ top_line }}
+{{ top_underline * ((top_line)|length)}}
+{% elif versiondata.name %}
+{{ versiondata.name }} {{ versiondata.version }} ({{ versiondata.date }})
+{{ top_underline * ((versiondata.name + versiondata.version + versiondata.date)|length + 4)}}
+{% else %}
+{{ versiondata.version }} ({{ versiondata.date }})
+{{ top_underline * ((versiondata.version + versiondata.date)|length + 3)}}
+{% endif %}
+{% for section, _ in sections.items() %}
+{% set underline = underlines[0] %}{% if section %}{{section}}
+{{ underline * section|length }}{% set underline = underlines[1] %}
+
+{% endif %}
+
+{% if sections[section] %}
+{% for category, val in definitions.items() if category in sections[section]%}
+{{ definitions[category]['name'] }}
+{{ underline * definitions[category]['name']|length }}
+
+{% if definitions[category]['showcontent'] %}
+{% for text, values in sections[section][category].items() %}
+- {{ text }} ({{ values|join(', ') }})
```

Comment:
Can we somehow move the issue numbers at the end of the first line, instead of at the end of the entry? This looks a bit weird:
![image](https://user-images.githubusercontent.com/8530778/178112035-2a131d5b-162f-430b-8889-237fd5a2b0bf.png)

Reply:
This is a jinja2 template which its probably possible but I am not entirely sure how.

Reply:
https://github.com/DisnakeDev/disnake/pull/600/commits/2e51d29c1352bed7ab1d8b95b8616840aa81ac1e